OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -319,7 +322,7 @@ Action InitDisplayPanel(Widget panel)
 
 			var restartDesc = panel.Get("RESTART_REQUIRED_DESC");
 			restartDesc.IsVisible = () => ds.Mode != OriginalGraphicsMode || ds.VideoDisplay != OriginalVideoDisplay || ds.GLProfile != OriginalGLProfile ||
-				(ds.Mode == WindowMode.Windowed && (origWidthText != windowWidth.Text || origHeightText != windowHeight.Text));
+				(ds.Mode == WindowMode.Windowed && (origWidthText != windowWidth.Text || origHeightText != windowHeight.Text || ds.Language != OriginalLanguage));
```

Comment:
Won't this make it only restart in Windowed mode?

Reply:
You are absolutely right. I didn't notice as I always run in windowed mode.